### ISSUE TITLE
what is the best configuration in RNAN?

### ISSUE THREAD

**Issue by gittigxuy** [yulunzhang/RCAN#65]
In your RNAN paper,the loss is L2 loss,but the default is L1 loss.However,when I fix it to L2 loss,the result is not better than L1 loss and the L2 loss is extramely large.FurtherMore,the default setting of n_resgroups is the number of Resbolck layer,the default is 10,when I fix it to 20,the result is not good.So,how to fix the configuration?

**Comment by yulunzhang** [comment]
Hi,
You can just follow the specific configuration for each task in the provided training scripts.

**Comment by gittigxuy** [comment]
But from my experiment,the L1 loss is better than L2 loss,However ,in your paper,the RNAN use L2 loss

**Comment by yulunzhang** [comment]
It doesn't matter what the loss function (not affecting the main part of paper too much) we use.   

You may further obtain better results by using other improvements (like L1 loss as you said), if you like and could.

**Comment by gittigxuy** [comment]
so，how about the number of n_resgroups？the default is 10，if I increase the resgroup，will it be better？
